spec/system/composer/default_to_subcategory_spec.rb

```diff
@@ -6,32 +6,66 @@
   fab!(:user) { Fabricate(:user) }
   fab!(:group) { Fabricate(:group) }
   fab!(:group_user) { Fabricate(:group_user, user: user, group: group) }
+  fab!(:default_latest_category) { Fabricate(:category) }
   fab!(:category) { Fabricate(:private_category, group: group, permission_type: 3) }
   fab!(:subcategory) do
     Fabricate(:private_category, parent_category_id: category.id, group: group, permission_type: 1)
   end
+  fab!(:category_with_no_subcategory) do
+    category_with_no_subcategory = Fabricate(:private_category, group: group, permission_type: 3)
+    category_with_no_subcategory.read_restricted = false
```

Comment:
This reads a little odd to me because if we want a category that is not read restricted, we can just fabricate a normal category instead of a fabricating a private category and then updating the record to be not read restricted.